cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -261,3 +279,29 @@ func (guarantor *engineExistenceGuarantor) getRunningAndCLIEngineVersions() (*se
 
 	return runningEngineSemver, launcherEngineSemver, nil
 }
+
+func (guarantor *engineExistenceGuarantor) ensureCentralizedLogsComponentsAreRunning(ctx context.Context, shouldForceContainerRestart bool) error {
+
+	logsDatabase, err := guarantor.kurtosisBackend.GetLogsDatabase(ctx)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred getting the logs database")
+	}
+	isThereLogsDatabase := logsDatabase != nil
+	isThereNotRunningLogsDatabase := isThereLogsDatabase && logsDatabase.GetStatus() != container_status.ContainerStatus_Running
```

Comment:
Wait, isn't this the piece of code you're deleting in the other PR?? How do those two PR interact with each other?